### PR TITLE
win_irq_check: Remove 'IRQ' cases as no real value

### PR DESCRIPTION
**Pull request by menli820**
Remove 'IRQ' cases for all drivers as no real value.

ID: 2095144
Signed-off-by: Menghuan Li menli@redhat.com

### CODE REVIEW

**Comment by menli820** [comment]
@leidwang  @xiagao  @peixiu  @6-dehan  @kemama
Please help to review whether satisfy your feature requirement.

**Comment by xiagao** [comment]
Balloon part is ok. ACK

File: qemu/tests/cfg/win_irq_check.cfg [hunk 1]
```diff
@@ -84,21 +84,15 @@
...
                 - by_registry:
                     no with_balloon, with_viorng
                     msi_cmd = "reg add "HKLM\System\CurrentControlSet\Enum\%s\Device Parameters\Interrupt Management\MessageSignaledInterruptProperties" /v MSISupported /d %d /t REG_DWORD /f"
                     variants:
-                        - default_queues:
                         - multi_queues:
                             only with_netkvm
                             queues = 4
                 - by_vectors:
-                    only with_viostor, with_netkvm
```
Comment (peixiu) [inline qemu/tests/cfg/win_irq_check.cfg:99]: If only netkvm need this case, adjust the variants structure only support  netkvm is better.

File: qemu/tests/cfg/win_irq_check.cfg [hunk 2]
```diff
@@ -84,21 +84,15 @@
                     device_type = "virtio-tablet-pci"
                     input_dev_type_input1 = tablet
     variants:
-        - @default:
         - msi_disable:
```
Comment (peixiu) [inline qemu/tests/cfg/win_irq_check.cfg:87]: For vioscsi, both @default and msi_disable cases can be deleted, so reset this variants only consider the driver of need this case is ok.

Thanks~
Peixiu

**Comment by leidwang** [comment]
For vioinput, we can delete with_vioinput directly. Because we removed `default_queues` and vioinput doesn't need to test other scenarios.

**Comment by Kemama** [comment]
Hi,
After tests, LGTM. 

Ack: mama <mama@redhat.com>

**Comment by menli820** [comment]
> For vioinput, we can delete with_vioinput directly. Because we removed `default_queues` and vioinput doesn't need to test other scenarios.

@vivianQizhu  after feature owner review, we should only keep netkvm related case is okay,  do you think we should  also remove 
variants eg: - with_viorng:  - with_balloon .(actually remove @default can satisfy our requirement)

**Comment by vivianQizhu** [comment]
@menli820 I agree with @peixiu We should adjust the structure, remove those useless parameters.

**Comment by menli820** [comment]
@peixiu  @leidwang could you please help check it again? 

Now only netkvm related case can be filtered.
```
 (1/3) Host_RHEL.m9.u1.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: STARTED
 (1/3) Host_RHEL.m9.u1.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (2/3) Host_RHEL.m9.u1.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.win_irq_check.msi_disable.by_registry.multi_queues.with_netkvm.q35: STARTED
 (2/3) Host_RHEL.m9.u1.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.win_irq_check.msi_disable.by_registry.multi_queues.with_netkvm.q35: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (3/3) Host_RHEL.m9.u1.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.win_irq_check.msi_disable.by_vectors.with_netkvm.q35: STARTED
 (3/3) Host_RHEL.m9.u1.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.win_irq_check.msi_disable.by_vectors.with_netkvm.q35: CANCEL: Test cancelled due to --dry-run (0.00 s)
```

**Comment by menli820** [comment]
@peixiu  As we discussed, hace modified it, thanks 

```
 (1/2) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.win_irq_check.msi_disable_by_registry.q35: STARTED
 (1/2) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.win_irq_check.msi_disable_by_registry.q35: PASS (162.45 s)
 (2/2) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.win_irq_check.msi_disable_by_vectors.q35: STARTED
 (2/2) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.win_irq_check.msi_disable_by_vectors.q35: PASS (89.92 s)
```
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado/job-results/job-2022-06-23T05.25-964a7ea/results.html

**Comment by leidwang** [comment]
LGTM now.

**Comment by peixiu** [comment]
Looks good to me, thanks~

Acked-by: Peixiu Hou <phou@redhat.com>

**Review by peixiu — APPROVED** [review]

**Comment by menli820** [comment]
@vivianQizhu  please help to check it again when you free , thanks

**Comment by vivianQizhu** [comment]
Thanks all.

**Pull request merged**